pkg/permission/permission.go

```diff
@@ -61,7 +61,11 @@ func (ps *Permission) IsAllowed(ctx context.Context, action string, userID int,
 	for i, payload := range payloadList {
 		allowed, err := handler.IsAllowed(ctx, userID, payload)
 		if err != nil {
-			return false, fmt.Errorf("payload %d: %w", i, err)
+			bs, err := json.Marshal(payload)
```

Comment:
I'm no go-programmer, but to me it seems you would clear the content of **err**, but in line 68 you use **err** to return the error message from the handler.IsAllowed-method.

Reply:
Thank you very mutch. You are right. This is an ugly bug.

Reply:
Done. Please have another look